spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/ProcedureInput.java

```diff
@@ -153,7 +166,7 @@ private <K, V> Map<K, V> map(
     return convertedMap;
   }
 
-  public Identifier ident(ProcedureParameter param) {
+  public Identifier asIdent(ProcedureParameter param) {
```

Comment:
I'm not sure we actually want to use `asIdent` here, I think it makes sense for obtaining the other types but ident by  itself should be sufficient